PublicMapping/districtbuilder
# Style Login/Register/Forgot Password, buttons, links

## Overview
Styles the login, register, and forgot password flows. I've also added fonts (New Frank from the marketing site and Proxima Nova for better legibility within the application) and added additional theme styles for `Button`, links, etc.

### Demo
<img width="1792" alt="Screen Shot 2020-08-10 at 8 33 13 PM" src="https://user-images.githubusercontent.com/5672295/90020957-80b23300-dca8-11ea-9877-ec4c9a1140bd.png">

<img width="1792" alt="Screen Shot 2020-08-10 at 8 33 20 PM" src="https://user-images.githubusercontent.com/5672295/90020973-8871d780-dca8-11ea-92fd-cddda86c729a.png">

![Uploading Screen Shot 2020-08-12 at 12.08.10 PM.png…]()

<img width="1506" alt="Screen Shot 2020-08-11 at 3 04 00 PM" src="https://user-images.githubusercontent.com/5672295/90020889-60827400-dca8-11ea-9025-704fc03f76f1.png">

<img width="1678" alt="Screen Shot 2020-08-12 at 2 53 13 PM" src="https://user-images.githubusercontent.com/5672295/90023545-c290a880-dcab-11ea-866f-af566ca2b0f6.png">
<img width="950" alt="Screen Shot 2020-08-12 at 2 53 07 PM" src="https://user-images.githubusercontent.com/5672295/90023569-c7edf300-dcab-11ea-89e6-8bfe154ef0f1.png">


### Notes
- Jeff, Proxima Nova seemed like a good font for the app. Fairly low-key, but gives us the tabular formatting:
<img width="533" alt="Screen Shot 2020-08-12 at 2 34 20 PM" src="https://user-images.githubusercontent.com/5672295/90021268-f0282280-dca8-11ea-90d9-40d826209a67.png">

- I saw that some logos are in `public/`, but in reading [the React App docs](https://create-react-app.dev/docs/adding-images-fonts-and-files/) it seemed that its better to pull these in the way that I am (from that `media/` folder). I'm not confident though. Let me know if you'd prefer I do this a different way.
- I didn't change the error messages. I think we could make a new issue for this.
- I didn't use the `style` const at the top of the page here, but absolutely can. It didn't seem necessary for these bits but I'm very open (especially on the Password popover).

## Testing Instructions
- `git pull`
- Go to `http://localhost:3000/login`
- Note the new styles. Enter your username incorrectly to see an error on the input.
- Note hover, active and focus states for the buttons.
- Click "Sign up for free" link.
- Click on the Password input to see the popover. Change the screen width to see it move overtop of the input for less wide screens.
- Go through the Forgot Password flow http://localhost:3000/forgot-password

Closes #273
